src/templates/container-hotkey.jsx

```diff
@@ -1,53 +1,53 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
```

Comment:
`React` is the default export while `Component` and `PropTypes` are named exports. They have different semantics than `require()`, so it's not right to `import React` and then use `React.Component`.